android/app/src/androidTest/java/com/microsoft/codepush/common/managers/UpdateManagerAndroidTests.java

```diff
@@ -186,10 +186,11 @@ public void entryPathIsNullWhenPackageIsNull() throws Exception {
      */
     @Test
     public void verifyTest() throws Exception {
-        executeWorkflow(codePushUpdateManager, SIGNED_PACKAGE_HASH, SIGNED_PACKAGE_URL);
-        codePushUpdateManager.mergeDiff(SIGNED_PACKAGE_HASH, SIGNED_PACKAGE_PUBLIC_KEY, "index.html");
-        executeWorkflow(codePushUpdateManager, SIGNED_PACKAGE_HASH, SIGNED_PACKAGE_URL);
-        codePushUpdateManager.mergeDiff(SIGNED_PACKAGE_HASH, null, "index.html");
+        //TODO refactor test
```

Comment:
What is the purpose of this test, what are you trying to verify? Please provide a clarification here and below.

Reply:
@sergey-akhalkov All clarifications are present in javadocs for each of test methods.

Reply:
Ahh I see, thanks